Orange/widgets/visualize/owvenndiagram.py

```diff
@@ -399,7 +400,7 @@ def extract_columnwise(self, var_dict, columns=None):
                                                         self.atr_vals[atr_type])
                                                 .reshape(-1, 1))
                 else:
-                    new_atr = deepcopy(var_data[1][0][0])
+                    new_atr = var_data[1][0][0].copy()
```

Comment:
`deepcopy` used to be used here because `Variable.copy` hasn't existed yet. Now it does, and it should be safer than `deepcopy`.